test/smoke/testApps/build.gradle

```diff
@@ -121,5 +121,8 @@ subprojects {
 tasks.register("listTestApps") {
     group = "Help"
     description = "List all available test apps"
-    subprojects.collect { it.path }.each { println it }
+    doFirst {
```

Comment:
@trask doFirst works now.. curious why?

Reply:
I don't remember, I was able to change this back in the other PR too